WordPress/src/main/res/values/key_strings.xml

```diff
@@ -150,6 +151,26 @@
         <item>-1</item>
     </string-array>
 
+    <string-array name="weekdays">
```

Comment:
Do you know if there is a way to get the localized weekdays using any of the OS apis? Because, for example, in Turkey days start with Monday, so if there is an easy way to achieve that, I think that'd be something worth doing. If there is no easy way, we can just ignore it.